timesketch/tsctl.py

```diff
@@ -21,6 +21,7 @@
 import click
 from flask.cli import FlaskGroup
 from sqlalchemy.exc import IntegrityError
+from jsonschema import validate, ValidationError
```

Comment:
Do we need to add this to requirements.txt?

Reply:
Added as discussed.